pynumaflow/__init__.py

```diff
@@ -1,17 +1,36 @@
 import logging
 import os
+import sys
 
 if os.getenv("PYTHONDEBUG"):
     os.environ["PYTHONASYNCIODEBUG"] = "1"
 
 
+class StdoutFilter(logging.Filter):
+    """
+    Filter logs with level less than logging.ERROR so they will go to stdout instead
+    of default stderr
+    """
+
+    def filter(self, record: logging.LogRecord) -> bool:
+        return record.levelno in (logging.DEBUG, logging.INFO, logging.WARNING)
```

Comment:
Small nitpick. Lets use a set instead of a tuple. This will make this lookup in O(n).